components/FilterList/index.js

```diff
@@ -42,7 +42,7 @@ export default function FilterList({
       </StyledButtonWrapper>
       {filteredProjects.length === 0 ? (
         <StyledEmptyMessage>
-          "No {activeFilter} projects available!"
+          "No {activeFilter} projects available!";
```

Comment:
```suggestion
          "No {activeFilter} projects available!"
```

; is not needed here